docs/develop/sdk/overview.mdx

```diff
@@ -10,40 +10,39 @@ import TabItem from '@theme/TabItem';
 
 Restate offers a Java SDK ([sdk-java repo](https://github.com/restatedev/sdk-java)) and a TypeScript SDK ([sdk-typescript repo](https://github.com/restatedev/sdk-typescript)).
 
+:::info TS Handler API vs. gRPC API
+Restate currently supports two TypeScript APIs for implementing services: the Handler API and the gRPC API.
+The gRPC API requires you to specify the services, methods and data types via a Protobuf definition, as explained in the [service contract docs](/develop/service_type#restate-service-contract).
+The Handler API does not require this.
+:::
+
 ## Installation
 
 <Tabs groupId="sdk" queryString>
 <TabItem value="ts" label="TypeScript">
 
 :::tip
-To get started quickly, use the [template generator](https://github.com/restatedev/node-template-generator)
+Have a look at the [TypeScript Quickstart](/get_started/quickstart?sdk=ts)!
 :::
 
-
 Add the `@restatedev/restate-sdk` dependency to your NodeJS project to start developing Restate services.
 
-:::info
-Restate currently supports two TypeScript APIs for implementing services: the Handler API and the gRPC API.
-The gRPC API requires you to specify the services, methods and data types via a Protobuf definition, as explained in the [service contract docs](/develop/service_type#restate-service-contract).
-The Handler API does not require this.
-:::
-
 </TabItem>
 <TabItem value="java" label="Java">
 
 :::tip
-To get started, check out [the Java template project](https://github.com/restatedev/examples/tree/main/templates/java-gradle).
+Have a look at the [Java Quickstart](/get_started/quickstart?sdk=jvm)!
 :::
 
 Scaffold a project using the build tool of your choice. For example, with Gradle (Kotlin script):
 
-```
+```shell
```

Comment:
What's the difference between `shell` and `bash` syntax highlighting? For the `curl` command we replaced `shell` with `bash`. Should this be consistent?

Reply:
I think for the commands we have, there is no difference between the two. 
I replaced a few because I didn't see any highlighting, and I was debugging the issue. But the problem was that `bash` wasn't added to the list of languages to highlight with prism: https://github.com/restatedev/documentation/blob/0c509d77f768e287b83f6e90637bc1b64c6283b0/docusaurus.config.js#L168